### PR TITLE
Completed exercise unit-tests-jest.

### PR DESCRIPTION
**Pull request by j-alejandro-araujo**
Screenshot 2023-07-24 at 11 02 36 AM

### CODE REVIEW

File: unit-tests-jest/numbers.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+ * given divisor. Does not modify the original array.
+ * @param {Array} numbers The array of numbers to be divided by the divisor.
+ * @param {number} divisor The number to divide into each element of `numbers`.
+ * @returns a new array.
+ */
+export function divideBy(numbers, divisor) {
+  for (let i = 0; i < numbers.length; i++) {
+    numbers[i] = numbers[i] / divisor;
+  }
+  return numbers;
```
Comment (RobertGardner) [inline unit-tests-jest/numbers.js:29]: This modifies the original array, but spec says "Does not modify the original array."

File: unit-tests-jest/numbers.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+export function multiplyBy(obj, multiplier) {
+  const result = {};
+  Object.entries(obj).forEach(([key, value]) => {
+    if (typeof value === 'number') {
+      result[key] = value * multiplier;
+    } else {
+      result[key] = value;
+    }
+  });
+  return result;
```
Comment (RobertGardner) [inline unit-tests-jest/numbers.js:48]: This does not modify original object but spec says "Modifies an object".

File: unit-tests-jest/numbers.test.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import { evenNumbers, toDollars, divideBy, multiplyBy } from './numbers';
+
```
Comment (RobertGardner) [inline unit-tests-jest/numbers.test.js:2]: These tests all test the happy path nicely, but you should also test the edge cases to make sure you handle them properly.

**Review by RobertGardner — APPROVED** [review]
